checklist.md

```diff
@@ -0,0 +1,50 @@
+# This is the requirement checklist for an `azd` template to be considered complete
+
+# Repository Management
+ 
+- [ ] **README** - Standards compliant [README.md](../../README.md) as the one in the example, is in place
```

Comment:
Is the idea that additional README requirements like the following would go in the README.md, rather than being called out in this checklist?

- [ ] **Prerequisites** - a Prerequisites section should be included. [Example](https://github.com/Azure-Samples/todo-python-mongo-aca#prerequisites)

- [ ] **Architecture** - an architecture diagram and description must be included. [Example](https://github.com/Azure-Samples/todo-python-mongo-aca#application-architecture)

- [ ] **'Open In __' buttons** - must use same "Open in " buttons as the TODO samples. [Example](https://github.com/Azure-Samples/todo-python-mongo-aca/blob/05a7aa59c0253628e293ca0fcc98f35a942df1cc/README.md?plain=1#L24)

- [ ] **Cost Estimation** - a Cost Estimation section should be included. [Example](https://github.com/Azure-Samples/azure-search-openai-demo#cost-estimation)

Reply:
Yes, we can say that explicitly @charris-msft.